src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/NativeScrollContentPresenter.Android.cs

```diff
@@ -55,6 +55,7 @@ public NativeScrollContentPresenter()
 			SetForegroundGravity(GravityFlags.Fill);
 
 			SetClipToPadding(false);
+			SetClipChildren(false);
```

Comment:
What does this change? Is it possible to add a test for it?

Reply:
Indeed, automated tests to validate this are needed.